controller/controller_manager.go

```diff
@@ -40,6 +45,8 @@ func StartControllers(stopCh chan struct{}, controllerID, serviceAccount, manage
 	if err != nil {
 		return nil, nil, errors.Wrap(err, "unable to get client config")
 	}
+	config.QPS = QPS
```

Comment:
If we remove the blocking call from reconciliation process, do we still need these? I am still not comfortable with tweaking those parameters without knowing the potential downside.